include/fi_util.h

```diff
@@ -376,12 +373,18 @@ int ofi_av_get_index(struct util_av *av, const void *addr);
  * Connection Map
  */
 
+// TODO explore replacing this with a simple connection hash map that is common
+// for both AV and RX only connections.
 #define UTIL_CMAP_IDX_BITS 48
 
 enum util_cmap_state {
 	CMAP_UNSPEC,
-	CMAP_CONNECTING,
-	CMAP_CONNECTED
+	CMAP_CONN_INIT,
+	CMAP_CONNECT,
```

Comment:
I'm not sure that's clearer.  The states below: CONNREQ and ACCEPT are also part of "connecting".  Does this state mean CONNREQ_SENT?
On this note, without looking at the code, it's not clear why there's a CONN_INIT state.  That state seems like it be immediately transitioned into actively connecting.